SA3/qa_web/templates/qa_web/includes/editor_style.html

```diff
@@ -1,28 +1,28 @@
 {% load static %}
```

Comment:
Still weirded out by the existence of this file, but it is necessary for the display of buttons on the MD Editor.